### PR TITLE
Build for aarch64

### PR DESCRIPTION
**Pull request by hfiguiere**
Build kdenlive on aarch64, including with changes for ffmpeg and fftw3f.

I have tested summarily on Raspberry Pi 400 running the 64-bits OS. (and built on it)

Also in separate commits:

- Fix a warning in the manifest syntax
- Change two url of dependencies as the original are 404. Not sure if there are better options. Eigen moved to gitlab and the new tarball has a different sha256. x265, I located a mirror and the checksum is unchanged.

### CODE REVIEW

**Comment by hfiguiere** [comment]
I forgot to enable aarch64 on flathub.

File: flathub.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "only-arches": ["x86_64"]
```
Comment (tsdgeos) [inline flathub.json:2]: i think you could remove this altogether since these are the two arches that are build in flathub anyhow at the moment, but not important, let's merge this and if we feel like removing this file, we can do it in a separate PR
